osu.Game/Screens/Backgrounds/BackgroundScreenBeatmap.cs

```diff
@@ -49,14 +52,31 @@ public WorkingBeatmap Beatmap
             }
         }
 
+        public Color4 DimColour
+        {
+            get { return dimColourBox.Colour; }
+            set {
+                if (dimColourBox != null) dimColourBox.Colour = value;
+            }
+        }
+
         public BackgroundScreenBeatmap(WorkingBeatmap beatmap = null)
         {
             Beatmap = beatmap;
+            Add(dimColourBox = new Box
```

Comment:
As I mentioned in the issue, this is adding 1x overdraw. At *very least*, this needs to be inside the `BufferedContainer` used to render the blur as to avoid overdraw in every gameplay frame, but even then, it has the potential to slow down (already GPU heavy) transitions to a crawl.

Reply:
I tried moving it to the BeatmapBackground but couldn't get the dim level FadeTo work properly since CacheDrawnFrameBuffer is true. This is probably something I can't fix since my knowledge of OpenGL/GLSL is very limited.

Reply:
You'd need to call `ForceRedraw` or whatever the function is called to refresh the buffer

Reply:
Looks like you're doing something very wrong.

Reply:
Yeah that is my guess too 😅